Tests/Controller/Annotations/AbstractScalarParamTest.php

```diff
@@ -55,6 +58,22 @@ public function testComplexRequirements()
         ), $this->param->getConstraints());
     }
 
+    public function testMultipleComplexRequirements()
+    {
+        $requirement1 = $this->getMockBuilder('Symfony\Component\Validator\Constraint')->getMock();
```

Comment:
let's use the `class` constant and a `use` statement for the class

Reply:
The original code already used FQCN strings so I just did the same. Anyway, fixed.